common/styles/scss/_navbar.scss

```diff
@@ -17,7 +17,7 @@ body > .container {
 }
 
 .error-open #mainnav {
-    z-index: 1051;
+    z-index: 1501;
```

Comment:
Now that you know why you're using this specific value. Can you please comment on the reason for it here as well?